### PR TITLE
Descriptive 404

### PR DESCRIPTION
**Pull request by svbeon**
This is an attempt at a compromise on redirects for outdated links which could be both costly and technically problematic, this solutions adds more information or the 404 page in an order inspired by @bigpresh

### CODE REVIEW

**Comment by shadowcat-mst** [comment]
Needs http://freenode.net/kb/answer/registration linking as well, otherwise nice.

Also it would be nice if we replaced "you've followed an outdated link" with something that isn't accusatory in tone.

File: content/pages/404.md [hunk 1]
```diff
@@ -4,8 +4,25 @@ slug: 404
 absolute: yes
 ---
 
-You've followed an outdated link (sorry) or otherwise ended up at a page that
-doesn't exist.
+This page doesn't exist. Our [new website](/news/new-website) might be the
+cause, thus old links won't work anymore. If you followed this link from
+an external source we would reccomend reporting it to them. If you
```
Comment (equinox) [inline content/pages/404.md]: Typo: reccomend

Comment (svbeon) [inline content/pages/404.md]: Fixed

**Comment by bigpresh** [comment]
Looks good to me!

**Pull request merged**